code_examples/sdk_examples/src/core_features/did/11_full_did_delete.ts

```diff
@@ -10,9 +10,8 @@ export async function deleteFullDid(
   // Create a DID deletion tx. We specify the number of endpoints currently stored under the DID because
   // of the upper computation limit required by the blockchain runtime.
   const didIdentifier = Kilt.Did.toChain(fullDid)
-  const endpointsCountForDid = await api.query.did.didEndpointsCount(
-    didIdentifier
-  )
+  const endpointsCountForDid =
+    await api.query.did.didEndpointsCount(didIdentifier)
```

Comment:
How come this isn't caught with the linting?

Reply:
@Dudleyneedham I did… But… I didn't actually change any of these files, and despite multiple actions runs (take a look) that asked me to run style and lint fixes on code that I also never touched, the tests all show as passed and nothing identifies this particular issue 🤷 .

So I fixed it manually.

Reply:
Ok fair enough.